benchmark/clblast/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CLBLAST_BENCH_SRCS
 foreach(clblast_benchmark ${CLBLAST_BENCH_SRCS})
   get_filename_component(clblast_bench_exec ${clblast_benchmark} NAME_WE)
   add_executable(bench_clblast_${clblast_bench_exec} ${clblast_benchmark} ${CLBLAST_BENCH}/main.cpp ${CLBLAST_BENCH}/clwrap.cpp)
-  target_link_libraries(bench_clblast_${clblast_bench_exec} PRIVATE benchmark ${BLAS_LIBRARIES} clblast sycl_blas)
+  target_link_libraries(bench_clblast_${clblast_bench_exec} PRIVATE benchmark blas::blas clblast sycl_blas)
```

Comment:
does clblast need to link with sycl_blas?

Reply:
See above.